app/models/Extraction.scala

```diff
@@ -138,6 +139,7 @@ object ExtractorInfo {
       (JsPath \ "external_services").read[List[String]].orElse(Reads.pure(List.empty)) and
       (JsPath \ "libraries").read[List[String]].orElse(Reads.pure(List.empty)) and
       (JsPath \ "bibtex").read[List[String]].orElse(Reads.pure(List.empty)) and
+      (JsPath \ "maturity").read[String] and
```

Comment:
Adding `(JsPath \ "maturity").read[String].orElse(Reads.pure("Development"))` will make it so if current extractors don't have maturity in their definition it will have a default, otherwise it will throw an error. We might still have extractors that use this method to register.

Reply:
Noting that a default value is set above on line 93, but I can add the requested change if you still think it's necessary.

Reply:
Without this it will fail to post if you don't provide a `maturity` level.

Reply:
```suggestion
      (JsPath \ "maturity").read[String].orElse(Reads.pure("Development")) and
```